tests/unit_tests/test_server_client.py

```diff
@@ -1467,3 +1472,69 @@ async def test_batch_returns_run_id(app: FastAPI) -> None:
         assert len(run_ids) == 2
         for run_id in run_ids:
             assert _is_valid_uuid(run_id)
+
+
+@pytest.mark.asyncio
+async def test_feedback_succeeds_when_langsmith_enabled() -> None:
+    """Test the server directly via HTTP requests."""
+
+    with patch("langserve.server.ls_client") as mocked_ls_client_package:
```

Comment:
This works! I know you were looking for other ways to do this.

The two other ways:

1) fast api dependency management system -- it handles depednency injection 
https://fastapi.tiangolo.com/reference/dependencies/?h=depends
https://fastapi.tiangolo.com/advanced/testing-dependencies/

Can do something like:

```python
async def get_client() -> langsmith.Client():
    return langsmith.Client() # <-- Not sure if we want to instantiate a client for each request, if not we can keep a global

    @app.post(namespace + "/feedback")
    async def feedback(
        feedback_create_req: FeedbackCreateRequest, client: Annotated[langsmith.Client, Depends(get_client)]
    ) -> Feedback:
```

2) respones package to mock a response